spk/deluge/src/requirements-crossenv.txt

```diff
@@ -1,28 +1,21 @@
-##
-## required wheels that need cross compilation
-##
-
-# [deluge]
-#deluge==2.1.1        ==> pure
-# Latest commit on develop branch
-#git+https://github.com/deluge-torrent/deluge.git@develop#egg=deluge==2.1.1
-# Commit from March 1st 2025 on develop branch
-git+https://github.com/deluge-torrent/deluge.git@0b5addf58eac1f379ee1af83247d8dee0c1eae78#egg=deluge==2.1.1.dev136
```

Comment:
Can you keep this part please? It helps when needed to go back to it or doing git bisects so it can be easly uncommented and adjusted.  Likewise for `libtorrent`.  thnx.